ledger/participant-integration-api/src/main/scala/platform/store/dao/events/package.scala

```diff
@@ -51,6 +51,13 @@ package object events {
     * contiguous stretch of the input [[Source]]. Well suited to perform group-by
     * operations of streams where [[K]] attributes are either sorted or at least
     * show up in blocks.
+    *
+    * Implementation detail: this method _must_ use concatSubstreams instead of
+    * mergeSubstreams to prevent the substreams to be processed in parallel,
+    * potentially causing the outputs to be delivered in a different order.
+    *
+    * See: https://github.com/digital-asset/daml/pull/8336
```

Comment:
Please remove this reference, i.e., `git blame` will show the relevant merge commit.